nixos/platform/network.nix

```diff
@@ -163,10 +167,8 @@ in
 
     };
 
-    services.udev.extraRules = lib.concatMapStrings
-      (interface: ''
-        SUBSYSTEM=="net" , ATTR{address}=="${interface.mac}", NAME="${interface.physicalDevice}"
-        '') interfaces;
+    services.udev.initrdRules = interfaceRules;
```

Comment:
I'm surprised that this wouldn't be something like `initrdExtraRules` ... ? :)

Reply:
This *did* get renamed to `boot.initrd.services.udev.rules` somewhere between 21.05 and 23.05.